assets/models/system/Mistral-7B-Instruct-v0-2/description.md

```diff
@@ -0,0 +1,62 @@
+# **Model Details**
+The Mistral-7B-Instruct-v0.2 Large Language Model (LLM) is an instruct fine-tuned version of the Mistral-7B-v0.2.
+
+Mistral-7B-v0.2 has the following changes compared to Mistral-7B-v0.1:
+- 32k context window (vs 8k context in v0.1)
+- Rope-theta = 1e6
+- No Sliding-Window Attention
+
+For full details of this model please read our [paper](https://arxiv.org/abs/2310.06825) and [release blog post](https://mistral.ai/news/la-plateforme/).
+
+## Model Architecture
+
+This instruction model is based on Mistral-7B-v0.1, a transformer model with the following architecture choices:
+- Grouped-Query Attention
+- Sliding-Window Attention
+- Byte-fallback BPE tokenizer
+
+## Limitations
```

Comment:
Please refactor description to match sections described in [document](https://microsoftapc-my.sharepoint.com/:w:/g/personal/ayushmishra_microsoft_com/ETjbEy6poUhBsdysFKduH3IBmrdEe2gP2Yzza3p6COAAhA?e=mKdodm)

Reply:
Please take reference from bert-base-uncased model how description should be formatted

Reply:
Updated